.github/workflows/checkbox-tics.yml

```diff
@@ -0,0 +1,109 @@
+name: TICS Code Quality Analysis
+
+on:
+  schedule:
+    - cron: '00 19 * * *'
+  workflow_dispatch:
+ 
+permissions:
+  contents: read
+
+jobs:
+  TICS:
+    runs-on: ubuntu-latest
+    environment: TICS
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: main
```

Comment:
Did you use this to debug? If so remove it, I'm pretty sure cron dispatches on main and if we want to run this let us shoot ouselves in the foot!
```suggestion
```